Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44166.cs

```diff
@@ -121,7 +121,7 @@ public _44166MDP()
 
 			Master = new _44166Master();
 			Detail = new _44166Detail();
-		}
+        }
```

Comment:
Yeah, had to reinstall VS yesterday, and spaces is the default in VS. Thought I'd caught all of them, but I guess not.